apps/webapp/src/fetchers/staking.ts

```diff
@@ -24,6 +25,9 @@ const isDelegationBalance = (assetBalance: AssetBalance, identityKey: IdentityKe
   return bech32IdentityKey(identityKey) === matchGroups.bech32IdentityKey;
 };
 
+const getDelegationTokenBaseDenom = (validatorInfo: ValidatorInfo) =>
+  `udelegation_${bech32IdentityKey(getIdentityKeyFromValidatorInfo(validatorInfo))}`;
+
```

Comment:
I wonder if this should be with the getters? Not sure. 🤔

Reply:
Yeah, I'm actually planning on (sometime soon, when I have a break) writing a bunch of `asX` getters that transform objects into different types, such as `asBech32String` for identity keys. I'll punt that to a future PR though.